src/lib/viewers/controls/media/MediaSettingsControls/_styles.scss

```diff
@@ -0,0 +1,48 @@
+@import '../styles';
+
+$padding: 8px;
+
+@mixin bp-MediaSettingsMenuItem {
+    display: table-row;
+    height: 26px;
+    padding: $padding / 2;
+    overflow: hidden;
+    white-space: nowrap;
+    outline: 0 none; // will be overridden by our own
+    cursor: pointer;
+}
+
+@mixin bp-MediaSettingsMenuItem-item {
+    display: table-cell;
```

Comment:
We should try to use flexbox as much as possible, instead of table styling.

Reply:
Tried flexbox, but I think table styling is simpler. The current styling looks very similar to tabular data format, so I think table styling is a good choice in this case.